pyo3-macros-backend/src/frompyobject.rs

```diff
@@ -106,8 +106,8 @@ enum ContainerType<'a> {
     StructNewtype(&'a Ident),
     /// Tuple struct, e.g. `struct Foo(String)`.
     ///
-    /// Fields are extracted from a tuple.
-    Tuple(usize),
+    /// Fields are extracted from a tuple where the variant contains the list of extraction calls.
```

Comment:
I think it would be nice to structure this more similarly to the `Struct` variant, i.e. have an example in the summary and put the explanation of the variant content into a separate line.

Reply:
The example is already there two lines above it.

Reply:
Indeed, I misread. Sorry for noise!